### PR TITLE
dcos-3667 standardize storing of docopt specification

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

File: cli/dcoscli/data/help/main.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Command line utility for the Mesosphere Datacenter Operating
```
Comment (jsancio) [inline cli/dcoscli/data/help/main.txt]: Maybe we can name this file `dcos.txt`.

File: cli/dcoscli/main.py [hunk 2]
```diff
@@ -103,6 +62,15 @@ def _main():
         return analytics.wait_and_capture(subproc)[0]
 
 
+def _doc():
+    """
+    :rtype: str
+    """
+    return pkg_resources.resource_string(
+        'dcoscli',
+        'data/help/main.txt').decode('utf-8')
```
Comment (jsancio) [inline cli/dcoscli/main.py]: Let's change this if we name the help file `dcos.txt`.

File: cli/tests/integrations/test_dcos.py [hunk 3]
```diff
@@ -27,50 +27,9 @@ def test_default():
...
-
-    DCOS_SSL_VERIFY             If set, specifies whether to verify SSL certs
-                                for HTTPS, or the path to the certificate(s).
-                                Can also be configured by setting
-                                `core.ssl_config` in the config.
-"""
-
-    assert_command(['dcos', '--help'],
-                   stdout=stdout)
+    with open('tests/data/help/main.txt') as content:
```
Comment (jsancio) [inline cli/tests/integrations/test_dcos.py]: Let's change this to `.../dcos.txt` if we rename the help file for the root dcos command.

**Comment by jsancio** [comment]
LGTM. :shipit: !!

File: cli/dcoscli/data/help/main.txt (hunk 1 above)
Comment (tamarrow) [inline cli/dcoscli/data/help/main.txt]: done

**Pull request merged**